src/engraving/dom/note.cpp

```diff
@@ -3545,9 +3547,21 @@ const Note* Note::lastTiedNote() const
         if (std::find(notes.begin(), notes.end(), note->tieFor()->endNote()) != notes.end()) {
             break;
         }
-        if (!note->tieFor()->endNote()) {
+        auto next = note->tieFor()->endNote();
+        if (!next) {
             break;
         }
+        if (positionTickOffset > 0) { // this is for a repeat
+            auto chord = next->chord();
+            auto intervals = score()->spannerMap().findOverlapping(next->tick().ticks(), (next->tick() + chord->actualTicks()).ticks());
+            // find start of first volta
```

Comment:
Originally I had hoped to be able to find a specific volta based on which repeat this was being queried for.  But in fact currently it's irrelevant, it's really only notes tied into the *1st* volta that are issue - there is no way of tying notes into other voltas.  Basically, if you're playing a repeat, it has to assume that ties to notes in the 1st volta should be ignored.

Really the proper solution for that would allow actually creating multiple ties from the start note to notes in the various voltas, but that's a much bigger change.